run-tests.pl

```diff
@@ -321,12 +321,6 @@ sub _run_test
       Future->wait_any(
          $f_test,
 
-         $loop->delay_future( after => 2 )
```

Comment:
I didn't see enough value to be worth keeping.

What's it guarding against? What's it meant to be telling me? If a test has taken two seconds, I *know* I'm waiting on it.